### DISCUSSION TITLE
New computer and I can't find the same Kernel or load pandas

### DISCUSSION THREAD

**Discussion by nbol48** [earthlab-education/Earth-Analytics-2023-01-Intro#286] [! GitHub help]
I got a new computer (not sure if this is relevant) and when I try to load pandas as the first line of code, It's returning an error that says ModuleNotFound. I also can't use the same kernel as when I finished the initial LA analysis. Any help is appreciated!

**Comment by jsmi374** [comment]
Hey @nbol48,

Are you working with the Time Series: Flood Return Period Project, and having issues with the import code?

**Reply by nbol48** [reply]
Yep exactly @jsmi374

**Reply by jsmi374** [reply]
@nbol48 
I believe you need to split the cell into 2 cells for the import code to work. That is separate the Pandas from the first few lines of code that was already populated

**Comment by eculler** [comment]
@nbol48 my guess is that you are missing the .devcontainer.json file, which sets up the environment in your repositories. You won't have it in your portfolio repository, so I'd recommend copying it over from your LA assignment repo. You'll then need to rebuild your codespace or start a new one (make sure to commit and sync any changes you made!)

**Reply by nbol48** [reply]
@eculler - That did the trick, thank you!
